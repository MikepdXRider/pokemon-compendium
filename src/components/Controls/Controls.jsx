import React from 'react'
import Filter from '../Forms/Filter.jsx'
import Sort from '../Forms/Sort.jsx'

export default function Controls({ typeDataArr, setSelectedTypeStr, selectedTypeStr, setSelectedSortOrderStr, selectedSortOrderStr }) {
    return (
        <section>
            <Filter typeDataArr={typeDataArr} selectedTypeStr={selectedTypeStr} setSelectedTypeStr={setSelectedTypeStr} />
            
            {
              selectedTypeStr && <Sort setSelectedSortOrderStr={setSelectedSortOrderStr} selectedSortOrderStr={selectedSortOrderStr} optionsArr={['asc', 'desc']}/>
            }
        </section>
    )
}
