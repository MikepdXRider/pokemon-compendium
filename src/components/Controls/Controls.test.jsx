import { render } from "@testing-library/react";
import Controls from "./Controls.jsx";


it('matches snapshot', () => {
    const {container} = render(<Controls typeDataArr={['test1', 'test2', 'test3']}/>)

    expect(container).toMatchSnapshot()
});