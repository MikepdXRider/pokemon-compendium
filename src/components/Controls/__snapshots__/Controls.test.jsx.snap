// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <section>
    <label>
      Type
      <select>
        <option
          value="All"
        >
          All
        </option>
        <option
          value="test1"
        >
          test1
        </option>
        <option
          value="test2"
        >
          test2
        </option>
        <option
          value="test3"
        >
          test3
        </option>
      </select>
    </label>
  </section>
</div>
`;
